### DISCUSSION TITLE
Is there some ways I could use this tool to do load test for RabbitMQ AMQP 1.0?

### DISCUSSION THREAD

**Discussion by msl12** [rabbitmq/rabbitmq-perf-test#285] [General]
Hi, I installed the **rabbitmq_amqp1_0** plugin on RabbitMQ to enable AMQP 1.0. And I want to do some load tests for the AMQP 1.0 that RabbitMQ supports.

Since I heard that **rabbitmq-perf-test** was the most powerful load testing tool for RabbitMQ, and I tried it. But it seemed that **rabbitmq-perf-test** could only support AMQP 0-9-1.

Could you tell me how to do this? Or is there any other tools that could do such load tests?

Thanks a lot!

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
PerfTest only supports one protocol. There usually are other tools from different developers that offer similar features for other protocols.

**Reply by msl12** [reply]
Alright, thanks a lot
